tests/Integration/Database/EloquentModelDateCastingTest.php

```diff
@@ -20,6 +20,7 @@ public function setUp()
             $table->increments('id');
             $table->date('date_field')->nullable();
             $table->datetime('datetime_field')->nullable();
+            $table->datetime('datetime_diff_field')->nullable();
```

Comment:
Just curious...why add a new column for this `datetime_diff`? Couldn't you have run the tests against the regular `datetime` column?

Reply:
If you look at the casting at the bottom of the test in the TestModel1, I needed an extra one to be able to write a separate cast for this specific column so existing tests wouldn't break.

Reply:
Oh, true.